### PR TITLE
after upload scraped data that is returned is added to the cases data…

### PR DESCRIPTION
**Pull request by odst0016**
It is working kinda hacky code isnt as clean as it can be but cant be avoided at this point. !!!!!THIS IS NOT COMPLETE DO NOT TRUST THE DATA!!!!!

### CODE REVIEW

**Review by kristapants — APPROVED** [review]
Great PR, few lines but this is big. this is pushing the data scraped by DS to our table for feeding back to the user.

**Review by MattBokovitz1 — APPROVED** [review]
This code looks great and is a big addition. The .env.sample is added so others can test pdf uploads on their local to S3

File: api/upload/uploadRouter.js [hunk 1]
```diff
@@ -6,13 +6,27 @@ const { v4: uuidv4 } = require('uuid');
...
 const router = express.Router();
+const db = require('../cases/caseModel');
 require('dotenv').config();
 
 const s3 = new AWS.S3({
   accessKeyId: process.env.AWS_ACCESS_KEY,
   secretAccessKey: process.env.AWS_SECRET_KEY,
 });
 
+// body: {
```
Comment (ryan-hamblin) [inline api/upload/uploadRouter.js:17]: Please delete this commented-out code.

File: api/upload/uploadRouter.js [hunk 2]
```diff
@@ -29,13 +43,40 @@ const uploadFile = (fileName) => {
...
     axios
-      .post(
-        `http://bucket-test-dev.us-east-1.elasticbeanstalk.com/items/${fileName}`,
-        { name: fileName }
-      )
+      .post(`${process.env.DS_API}${fileName}`, { name: fileName })
       .then((res) => {
-        console.log('AwwwwWWW YASSSSS 👁️ 👄👁️ ');
-        console.log(res);
+        console.log('AwwwwWWW YASSSSS 👁️👄👁️ ');
```
Comment (ryan-hamblin) [inline api/upload/uploadRouter.js:48]: Please remove console.log's from main.

File: api/upload/uploadRouter.js [hunk 3]
```diff
@@ -29,13 +43,40 @@ const uploadFile = (fileName) => {
...
+          type_of_violence_experienced: 'Not Applicable',
+          applicant_indigenous_group: 'Not Applicable',
+          applicant_language: 'English',
+          applicant_access_to_interpreter: true,
+          applicant_perceived_credibility: false,
+        };
+
+        db.add(test)
+          .then(() => {
+            console.log(data);
```
Comment (ryan-hamblin) [inline api/upload/uploadRouter.js:74]: Please remove console.log from main

File: api/upload/uploadRouter.js [hunk 4]
```diff
@@ -29,13 +43,40 @@ const uploadFile = (fileName) => {
...
+            fs.unlinkSync(path.join(__dirname, 'uploads', `${fileName}.pdf`));
+          })
+          .catch((err) => {
+            console.log(err);
```
Comment (ryan-hamblin) [inline api/upload/uploadRouter.js:78]: Please handle error messages properly by sending them back to the client.

File: api/upload/uploadRouter.js [hunk 5]
```diff
@@ -29,13 +43,40 @@ const uploadFile = (fileName) => {
...
+          });
       })
       .catch((err) => {
         console.log(err);
```
Comment (ryan-hamblin) [inline api/upload/uploadRouter.js:82]: Please handle error messages properly by sending them back to the client.

**Review by ryan-hamblin — COMMENTED** [review]
Great work here. I see there are instances of commented-out code and a few console.logs that I don't think have a place in main. I recommend fixing those before merging into main.

File: api/upload/uploadRouter.js (hunk 2 above)
Comment (JusticeMatthew) [reply api/upload/uploadRouter.js:48]: We need these logs because they are FABULOUS

**Pull request merged**